package.json

```diff
@@ -17,5 +17,10 @@
     "bufferutil": "^4.0.5",
     "uniqid": "^5.4.0",
     "ws": "^8.4.0"
+  },
+  "devDependencies": {
+    "pscid": "^2.9.3",
+    "purescript-language-server": "^0.16.3",
```

Comment:
`purescript-language-server` and `purty` should already be available in the shell, provided by `spago2nix`. Was there an issue with them (we might want to try to fix that instead of getting them from NPM)?